src/Mod/PartDesign/Gui/TaskHoleParameters.cpp

```diff
@@ -838,6 +838,7 @@ void TaskHoleParameters::changedObject(const App::Document&, const App::Property
     else if (&Prop == &hole->Diameter) {
         ui->Diameter->setEnabled(true);
         updateSpinBox(ui->Diameter, hole->Diameter.getValue());
+        ui->HoleCutDiameter->setMinimum(hole->Diameter.getValue() + 0.1);

```

Comment:
This line is repeated on line 131 - I know that it is not much, but this is logic nonetheless - it would be easy to miss second place if for some reason we for example change the 0.1 into some other number.

1. Can you extract it to small private method? something `void updateHoleCutDiameterMinimum()` for example? This method then should be called from constructor and that line. 
2. 0.1 could be extracted into `constexpr` to get rid of magic number.

Reply:
I don't mind the current code.